spec/models/claims/mentor_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe Claims::Mentor, type: :model do
+  context "with associations" do
+    it { is_expected.to have_many(:mentor_memberships) }
+    it { is_expected.to have_many(:schools).through(:mentor_memberships) }
+  end
+
+  describe "default scope" do
+    it "is scoped to providers using the placement service" do
+      mentor = create(:claims_mentor)
+      create(
+        :mentor_membership,
+        :claims,
+        mentor:,
+      )
+
+      expect(described_class.all).to contain_exactly(mentor)
+    end
```

Comment:
It'd be good to include a mentor which _should not_ appear in the results, too. Otherwise we're not demonstrating that non-claims mentors are filtered out by the default scope.

Here's an example of how a scope has been tested elsewhere:

https://github.com/DFE-Digital/itt-mentor-services/blob/9f7555d0065622ac4b3e97ade992b2251b6beb08/spec/models/provider_spec.rb#L55-L63

Reply:
c06a141d8659b694ad202fd79569d29eaeceaf2c